### ISSUE TITLE
[Feature Request] Provide example of evaluator_config for evaluate

### ISSUE THREAD

**Issue by pamelafox** [microsoft/promptflow#3203]
I suspect that I need to specify evaluator_config for evaluate in order to map the data from the target response, but there's no example of it in the docstring or in https://pypi.org/project/promptflow-evals/0.2.0.dev0/

Can you provide an example either in the docstring or the promptflow-evals docs?

**Comment by pamelafox** [comment]
I don't believe any more input is needed from me. (The stalebot in this repo seems to be tuned to respond really quickly to perceived stalneness, btw).

**Comment by luigiw** [comment]
@diondrapeck , can you pick this up while you're working on docs improvement? Thank you.
